src/ssl.rs

```diff
@@ -196,8 +198,18 @@ impl<IOCB: IOCallbacks> Session<IOCB> {
     /// Gets the current cipher of the session.
     /// If the cipher name is "None", return None.
     pub fn get_current_cipher_name(&self) -> Option<String> {
+        // SAFETY: [`wolfSSL_get_current_cipher`][0] ([also][1]) needs valid `wolfssl_sys::WOLFSSL` pointer as per documentation
+        // Return value is the pointer inside the ssl session. Caller can read it safely
+        //
+        // [0]: https://www.wolfssl.com/documentation/manuals/wolfssl/group__IO.html#function-wolfssl_get_current_cipher_suite
```

Comment:
linking to `wolfssl_get_current_cipher_suite` but the code is using `wolfssl_get_current_cipher`